tests/php/Services/Tabbar/TabbarService/TabbarServiceGetTabbarInfoTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+use App\Http\Services\Tabbar\TabbarService;
+use Illuminate\Http\Request;
+use App\Http\Consts\UrlInfo;
+
+/**
+ * TabbarServiceクラスのgetTabbarInfoのテストを行う.
+ *
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class TabbarServiceGetTabbarInfoTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)